Exonoesis/Helping-Hand
# Pull Request Template for Streamlined Collaboration

## What is the purpose of these changes?
This inclusion for a Pull Request template is to ensure that it is easier for maintainers to understand the motivation behind incoming changes. In addition, it maintains a long term record of intentions over the span of the whole project.

## What has changed?
Added a template for pull requests using markdown syntax.

## How can I verify this works?
1. Merge this Pull Request.
2. Make a new Pull Request.
3. Observe that the description field is pre-filled with this template.
